src/wp-includes/option.php

```diff
@@ -2105,16 +2097,8 @@ function update_network_option( $network_id, $option, $value ) {
 	 */
 	$value = apply_filters( "pre_update_site_option_{$option}", $value, $old_value, $option, $network_id );
 
-	/*
-	 * If the new and old values are the same, no need to update.
-	 *
-	 * Unserialized values will be adequate in most cases. If the unserialized
-	 * data differs, the (maybe) serialized data is checked to avoid
-	 * unnecessary database calls for otherwise identical object instances.
-	 *
-	 * See https://core.trac.wordpress.org/ticket/44956
-	 */
-	if ( $value === $old_value || maybe_serialize( $value ) === maybe_serialize( $old_value ) ) {
+	// If the new and old values are the same, no need to update.
+	if ( false !== $old_value && _is_equal_database_value( $old_value, $value ) ) {
```

Comment:
See above.

Reply:
Removed the `update_network_option` changes on this PR.